rest/rest-client/src/main/java/org/ow2/proactive/scheduler/rest/SchedulerClient.java

```diff
@@ -1515,17 +1519,60 @@ public boolean checkJobPermissionMethod(String jobId, String method) throws Sche
     }
 
     @Override
-    public Set<String> addJobSignal(String jobId, String signal)
+    public Set<String> addJobSignal(String jobId, String signal, Map<String, String> outputVariables)
```

Comment:
there are still many occurrences of outputVariables, can you please change them?